### ISSUE TITLE
autosync 

### ISSUE THREAD

**Issue by rocky-III** [x0b/rcx#202]
just discovered rcx today and i really like it... what I  missing is autosync of folders. It would be could if folders could stay in sync without that the process has to be triggered over and over again. Thanks

**Comment by jaylex32** [comment]
I believe this project is dead! We need a alternative for it.

**Comment by chymian** [comment]
@rocky-III
> just discovered rcx today and i really like it... what I missing is autosync of folders. It would be could if folders could stay in sync without that the process has to be triggered over and over again. Thanks

have a look at syncthing.net. it does exactly that, - keeps folder in sync.

**Comment by rocky-III** [comment]
@chymian

does syncthing.net work with servers and WebDav? Don`t think so.

**Comment by chymian** [comment]
@rocky-III
no, it uses its own protocol to keep directorys in sync between any devices.
if you need webdav, it's not for that.

Am 21. März 2022 17:02:59 GMT+07:00 schrieb rocky-III ***@***.***>:
***@***.***

**Comment by 31337-4554551n** [comment]
I would also very much like to request this.
I am looking for an open source alternative to foldersync.

Not all apps support SAF. It would be nice to use something like this as a middleman.
It already supports connecting to SAF shares, or even webdav by itself.
And allows syncing from those remotes to local folders for other apps to access.

I would love to see a constant sync between a local and remote (or SAF) folder. Allowing other apps to point to the local folder and this to play middleman. 

Would be happy to donate to make this happen.

**Comment by gotjoshua** [comment]
*IF* this
> [project is dead! We need](https://github.com/x0b/rcx/issues/202#issuecomment-1039575236)

to revive it!

**Comment by newhinton** [comment]
@31337-4554551n @rocky-III I have opened a PR for this, but it wasnt merged. I have forked it however: [extRact](https://github.com/newhinton/extRact)
